src/flash/options.ts

```diff
@@ -37,4 +37,8 @@ module Shumway.AVM2.AS {
   export var flvOption = flashOptions.register (
     new Shumway.Options.Option(null, "FLV support.", "string", "unsupported", "Defines how to deal with FLV streams.")
   );
+
+  export var scriptsTimeout = flashOptions.register (
+    new Shumway.Options.Option("", "Scripts timeout", "number", 1000, "Specifies script execution timeout before hang is declared (in ms).")
+  );
```

Comment:
Isn't 1 second a little bit low? The SWF spec recommends something from 5 to 20 seconds. Also, we might want to take the ScriptTimeoutSeconds in ScriptLimits tags (if present) into account.